app/src/main/java/ch/epfl/culturequest/social/Profile.java

```diff
@@ -57,12 +97,69 @@ public String getPhoneNumber() {
         return phoneNumber;
     }
 
-    public Uri getProfilePicture() {
+    public String getProfilePicture() {
         return profilePicture;
     }
 
-    public void updateProfilePicture(Uri profilePicture) {
+    public void setUid(String uid) {
+        this.uid = uid;
+
+    }
+
+    public void setName(String name) {
+        this.name = name;
+
+    }
+
+    public void setUsername(String username) {
+        this.username = username;
+
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }
```

Comment:
same for this. a user cant change their email unless they signout and sign in with a different account